app/models/site_setting.rb

```diff
@@ -191,6 +191,15 @@ def self.Upload
     SiteSetting::Upload
   end
 
+  class Whispers
+    def self.allowed_group_ids
+      SiteSetting.whispers_allowed_groups.present? ? SiteSetting.whispers_allowed_groups.split("|").map(&:to_i) : []
```

Comment:
Do we need to check whether whispers have been enabled? If whispers are not enabled, this should just return an empty array.